src/screens/SearchScreen.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from 'react';
+import { ScrollView } from 'react-native';
+
+import EditScreenInfo from 'components/EditScreenInfo';
+import { useEffect, useState } from 'react';
```

Comment:
you can combine these into one, eg. 
`import React, { useState, useEffect } from 'react'`